themes/classic/templates/catalog/_partials/product-add-to-cart.tpl

```diff
@@ -3,39 +3,47 @@
   {block name='product_quantity'}
     <div class="product-quantity">
       <div class="qty">
-        <input type="text" name="qty" id="quantity_wanted" value="{$product.quantity_wanted}" class="input-group">
+        <input
+          type="text"
+          name="qty"
+          id="quantity_wanted"
+          value="{$product.quantity_wanted}"
+          class="input-group"
+          min="{$product.minimal_quantity}"
+        >
       </div>
       <div class="add">
         <button class="btn btn-primary add-to-cart" data-button-action="add-to-cart" type="submit" {if !$product.add_to_cart_url}disabled{/if}>
           <i class="material-icons shopping-cart">&#xE547;</i>
     {l s='Add to cart' d='Shop.Theme.Actions'}
         </button>
         {block name='product_availability'}
-         {if $product.show_availability}
-            <span id="product-availability">
+          <span id="product-availability">
```

Comment:
Left as placeholder for replacement to work properly at https://github.com/thierrymarianne/PrestaShop/blob/5b9ce520cb4adeaaa7f43a2282ba6e5a4efcb9f8/themes/_core/js/product.js#L20